man/stopTrial.Rd

```diff
@@ -207,15 +207,6 @@ dose
   data = ANY
 )}: Stop based on probability of targeting biomarker
 
-\item \code{stopTrial(
```

Comment:
maybe good to run "Build > Document" again in RStudio to see if this should really be removed